portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['I like to rock climb!', 'I play Overwatch', 'I like to play guitar', 'I am Indian and Mexican!'];
 
   // Pick a random greeting.
```

Comment:
Consider changing this constant and the comment above it to facts since its no longer greetings.

Reply:
Smart! I'll rename everything in the function accordingly.